NEWS.md

```diff
@@ -1,6 +1,7 @@
 ## 2020-10-29 v1.10.0-SNAPSHOT
 * [MODDICONV-152](https://issues.folio.org/browse/MODDICONV-150) Upgrade to RMB v31.1.5
 * [MODDICONV-156](https://issues.folio.org/browse/MODDICONV-156) Disable CQL2PgJSON & CQLWrapper extra logging in mod-data-import-converter-storage.
+* [MODDICONV-148](https://issues.folio.org/browse/MODDICONV-148) Create default OCLC Create Instance profile.
```

Comment:
## xxxx-xx-xx v1.10.0-SNAPSHOT 
* [MODDICONV-148](https://issues.folio.org/browse/MODDICONV-148) Create default OCLC Create Instance profile

## 2020-11-06 v1.9.2
* [MODDICONV-152](https://issues.folio.org/browse/MODDICONV-150) Upgrade to RMB v31.1.5
* [MODDICONV-156](https://issues.folio.org/browse/MODDICONV-156) Disable CQL2PgJSON & CQLWrapper extra logging in mod-data-import-converter-storage

Reply:
Done